akka-remote/src/main/scala/akka/remote/artery/FlightRecorder.scala

```diff
@@ -208,12 +200,33 @@ private[remote] object FlightRecorder {
   val HiFreqEntryCountFieldOffset = 16
 }
 
+private[akka] object FlightRecorderExtension extends ExtensionId[FlightRecorder] with ExtensionIdProvider {
+
+  override def lookup(): FlightRecorderExtension.type = FlightRecorderExtension
+
```

Comment:
add the `def get(system: ActorSystem): Cluster = super.get(system)` also (for Java API), so that it's not forgotten in case we make this public later